x-pack/test/functional/services/ml/api.ts

```diff
@@ -299,6 +309,27 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
       });
     },
 
+    async waitForJobMemoryState(
```

Comment:
Nit - should this be `waitForJobMemoryStatus`? Message below and in the UI refers to it as `status`.

Reply:
renamed in https://github.com/elastic/kibana/pull/141775/commits/3aeb8c8832f8c7dac05f80badc803b3db0b2b49b